bodhi/server/scripts/check_policies.py

```diff
@@ -24,10 +24,15 @@
 messages in the message bus yet.
 """
 import click
+import logging
```

Comment:
For [PEP-8](https://www.python.org/dev/peps/pep-0008/#imports), ```logging``` should be imported in its own group since it is part of the standard library, and that group should come first.